sormas-ui/src/main/java/de/symeda/sormas/ui/samples/SampleController.java

```diff
@@ -238,7 +238,8 @@ private void requestSampleCollectionTaskCreation(SampleDto dto, SampleEditForm f
 			@Override
 			public void buttonClick(ClickEvent event) {
 				popupWindow.close();
-				ControllerProvider.getTaskController().createSampleCollectionTask(TaskContext.CASE, dto.getAssociatedCase(), dto);
+				ControllerProvider.getTaskController().createSampleCollectionTask(TaskContext.CASE,
```

Comment:
The task context also needs to change to TaskContext.CONTACT if the sample is only assigned to a contact